_posts/2021-03-15-Moving-Monsters.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Moving Monsters 
+description: 
+summary: 
+tags:
+---Today, we started off with typing club once again. Then, Kajari instructed us to work on our game that we have been working on for the past weeks. Today, we had to place both monsters on each side of the top of the canvas. We had to place monster one on the top left side whose coordinates are 0,0. This is because it is the top of the canvas and the coordinates have not progressed which is why it is 0,0. Now, for monster number 2, the coordinates are 448,0. This is because the width stays the same while the height increases, because we want monster 2 to the right side of the canvas. Below I have included a sketch of what kajari provided for me as a reference.
+```  
+    y                                                                                   
+x   0 -----------------------------------------------------------------------------------------canvas.width = 512                                                                 Monster(448,0)            |
+    |                                                                                    |      |
+    |                                                                                    |      |
+    |                                                                                    |______|
+    |                                                                                           |
+    |                                                                                           |
+    |                                                                                           |
+    |                                                                                           |
+    |                                              Hero(256,240)                                |
+    |                                               _______                                     |
+    |                                               |     |                                     |
+    |                                               |     |                                     |
+    |                                               |_____|                                     |
+    |                                                                                           |
+    |                                                                                           |
+    |                                                                                           |
+    |                                                                                           |
+canvas.height = 480-----------------------------------------------------------------------------
+```    
```

Comment:
Below the image explain about the fact that the x and y coordinates that you use to place the hero and the monster are the top left corners of the hero image and the monster image. So you have to take away the width of the monster from the width of the canvas in order for the monster to show on the right hand side. Otherwise it gets stuck in the white space to the right of the canvas.